osu.Game/Rulesets/UI/RulesetContainer.cs

```diff
@@ -282,6 +290,9 @@ private void applyMods(IEnumerable<Mod> mods)
 
             foreach (var mod in mods.OfType<IApplicableToRulesetContainer<TObject>>())
                 mod.ApplyToRulesetContainer(this);
+
+            foreach (var mod in mods.OfType<IReadFromConfig>())

```

Comment:
Shouldn't it read from config before applying to anything as it could change the outcome?